src/backend/distributed/executor/citus_custom_scan.c

```diff
@@ -705,19 +705,7 @@ CitusEndScan(CustomScanState *node)
  */
 static void
 CitusReScan(CustomScanState *node)
-{
-	CitusScanState *scanState = (CitusScanState *) node;
```

Comment:
I remember this discussion: https://github.com/citusdata/citus/pull/2635#discussion_r268079013

Does it work nowadays?

Reply:
yes, it seems to work fine now. It may have been related to real-time executor.